### PR TITLE
Handle multiple dimension case for BatchNormalization

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

**Comment by jiafatom** [comment]
Will use [this](https://github.com/onnx/keras-onnx/pull/106)
